### PR TITLE
macOS on M1

### PR DESCRIPTION
**Pull request by santoszv**
Tested on:
macOS Monterey
Version 12.4
Chip Apple M1

Observations:
It does not compile with "org.gradle.caching=true" in "gradle.properties"

### CODE REVIEW

**Comment by nedtwigg** [comment]
I compiled this locally with `./gradlew assemble` and it worked great. Thanks!

**Comment by Frotty** [comment]
@karlsabo Can you take a look at this?

File: PackrLauncher/packrLauncher.gradle.kts [hunk 1]
```diff
@@ -82,7 +82,7 @@ tasks.withType(CppCompile::class).configureEach {
 
 dependencies {
    implementation(project(":DrOpt"))
-   testImplementation("org.gradle.cpp-samples:googletest:1.9.0-gr4-SNAPSHOT")
+   //testImplementation("org.gradle.cpp-samples:googletest:1.9.0-gr4-SNAPSHOT")
```
Comment (karlsabo) [inline PackrLauncher/packrLauncher.gradle.kts:85]: Should we delete this line? Did Gradle drop support for googletest?

**Review by karlsabo — APPROVED** [review]
Seems fine. I don't have a mac to test it on.

**Comment by santoszv** [comment]
I do not remember why I included Google Test source code, instead of using it as a depenency. Today is 2024 and the commit is dated 2022, maybe there is a recent version and included source code is outdated.

Also, there are some changes in build scripts for supporting M1 chip, maybe a recent version of Gradle do not need this changes.

**Pull request merged**